routingv8/request.go

```diff
@@ -46,6 +46,12 @@ type CalculateMatrixRequest struct {
 	Body *CalculateMatrixBody
 }
 
+type RoutesRequest struct {
+	Origin        GeoWaypoint `json:"origin"`
+	Destination   GeoWaypoint `json:"destination"`
+	TransportMode string      `json:"transportMode"`
```

Comment:
Similar to how [CalculateMatrix does it](https://github.com/einride/here-go/blob/master/routingv8/request.go#L34), what do you think about using the [TransportMode enum](https://github.com/einride/here-go/blob/master/routingv8/request.go#L272) in the request?
That would align with the matrix usage, and give the developer a hand in understanding which transport modes are correct ones.

We could then use the TransportMode's String() method in the routes.go file to resolve the field to a string.